spec/request/users_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    before(:example) { get '/users' }
```

Comment:
- [ ] kindly test for a user's show action as well, just like you have done for the post controller actions.